k-distribution/src/main/scripts/bin/kparse-gen

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+set -u
+
+export PATH="$(cd "$(dirname "$0")"; pwd):$PATH"
+
+# initialize flags
+ARGV=()
+dir=.
+literal=false
+result=1
+verbose=false
+bisonFiles=()
+stackDepth=10000
+glr=--gen-parser
+
+# setup temp files
+now=`date +"%Y-%m-%d-%H-%M-%S"`
+tempDir="$(mktemp -d .kparse-gen-${now}-XXXXXXXXXX)"
+tempFiles=("$tempDir")
+trap 'rm -rf ${tempFiles[*]}' INT TERM EXIT
+
+execute () {
+  (
+  if $verbose; then
+    set -x
+  fi
+  "$@"
+  )
+}
+
+KPARSE_AOT=$(basename "$0")
+
+fold_lines="fold -s"
+
```

Comment:
Can we define this with the other initialize flags? Or is it here because this is the only use? I still may prefer to have it with the other flags.